docs/user/reference/log.md

```diff
@@ -52,7 +52,15 @@ In a controller high availability scenario, `logsink.log` is not guaranteed to c
 
 ### The machine-lock log file
 
-The machine-lock log file (`machine-lock.log`) contains logs for the file lock that synchronises hook execution on Juju {ref}`machines <machine>`. (A machine will only ever run one {ref}`hook` at a time.)
+The machine-lock log file (`machine-lock.log`) contains logs for the file lock that synchronises hook execution on Juju {ref}`machines <machine>`. (A machine will only ever run one {ref}`hook <hook>` at a time.)
+
+The purpose of this log file is to give more visibility to who has been holding the machine lock.
```

Comment:
todo: should add that the log file is only written AFTER the lock has been released. 

You can find the current lock holder via `juju_machine_lock` introspection tool